src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -24,26 +24,52 @@ export interface CrumbProps
       | "disabled"
     >,
     TagProps {
-  /** This sets the Crumb to current, does not render Link */
   isCurrent?: boolean;
+  onClick?: (
+    ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+  ) => void;
 }
 
-const Crumb = React.forwardRef<HTMLAnchorElement, CrumbProps>(
-  ({ href, isCurrent, children, onClick, ...rest }: CrumbProps, ref) => {
+const Crumb = React.forwardRef<HTMLLIElement, CrumbProps>(
+  ({ href, isCurrent, children, onClick, ...rest }, ref) => {
     const { isDarkBackground } = useBreadcrumbsContext();
 
+    const isSafari = React.useMemo(() => {
+      if (typeof navigator === "undefined") return false;
+      return /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+    }, []);
+
+    const handleClick = React.useCallback(
+      (
+        event: React.MouseEvent<
+          HTMLAnchorElement | HTMLButtonElement,
+          MouseEvent
+        >,
+      ) => {
+        if (!isCurrent && isSafari) {
+          (event.currentTarget as HTMLElement).focus({ preventScroll: true });
```

Comment:
suggestion: what you have is fine but if you use the ref (as long as it's passed to the anchor like before) you want have to cast the type to `HTMLElement`

```suggestion
          ref.current?.focus({ preventScroll: true });
```